### PR TITLE
Add section for parsec-cli-tests script in the quickstart guide

### PR DESCRIPTION
**Pull request by mohamedasaker-arm**
Relates: #534

Signed-off-by: Mohamed Omar Asaker <mohamed.omarasaker@arm.com>

### CODE REVIEW

File: src/getting_started/linux_x86.md [hunk 1]
```diff
@@ -75,6 +75,41 @@ MEYCIQCrc9cys35NeXwNAr8lYu8WPu0RiutkoAIWn+jYfYofPwIhAKPlNNsW//ykW8nX11KABNpWWYsN
 
 Execute the tool without any argument to check all the possible commands!
 
+## Running the `parsec-cli-tests.sh` script
+It's a script running simple end-two-end Parsec tests using parsec-tool and openssl
+
+As it uses the `parsec-tool`, `PARSEC_SERVICE_ENDPOINT` environment variable need to be set first.
+```
+export PARSEC_SERVICE_ENDPOINT=unix:$(pwd)/parsec.sock
```
Comment (anta5010) [inline src/getting_started/linux_x86.md:83]: Please add double quotes:
`export PARSEC_SERVICE_ENDPOINT="unix:$(pwd)/parsec.sock"`

File: src/getting_started/linux_x86.md [hunk 2]
```diff
@@ -75,6 +75,41 @@ MEYCIQCrc9cys35NeXwNAr8lYu8WPu0RiutkoAIWn+jYfYofPwIhAKPlNNsW//ykW8nX11KABNpWWYsN
...
+```
+
+Also `PARSEC_TOOL` environment variable should point to the parsec tool.
+```
+export PARSEC_TOOL=unix:$(pwd)/parsec-tool
```
Comment (anta5010) [inline src/getting_started/linux_x86.md:88]: double quotes as well and remove `unix:`

File: src/getting_started/linux_x86.md [hunk 3]
```diff
@@ -75,6 +75,41 @@ MEYCIQCrc9cys35NeXwNAr8lYu8WPu0RiutkoAIWn+jYfYofPwIhAKPlNNsW//ykW8nX11KABNpWWYsN
...
+```
+or the path of the `parsec-tool` is included in the `PATH` environment variable.
```
Comment (anta5010) [inline src/getting_started/linux_x86.md:90]: I would rephrase this paragraph. Something like:
If `parsec-tool` is not installed into a directory included in `PATH` then you also need to define `PARSEC_TOOL` environment variable with a full path to it:

export PARSEC_TOOL="\<full path\>/parsec-tool"

File: src/getting_started/linux_x86.md [hunk 4]
```diff
@@ -75,6 +75,41 @@ MEYCIQCrc9cys35NeXwNAr8lYu8WPu0RiutkoAIWn+jYfYofPwIhAKPlNNsW//ykW8nX11KABNpWWYsN
...
+
+- Test random number generation
+[DEBUG] Parsec BasicClient created with implicit provider "Mbed Crypto provider" and authentication data "UnixPeerCredentials"
+[INFO ] Generating 10 random bytes...
+[DEBUG] Running getuid
+[INFO ] Random bytes:
+A6 F5 90 24 DF FF 50 1F 29 2E
+....
+```
+
```
Comment (anta5010) [inline src/getting_started/linux_x86.md:109]: You can also include:

The parsec-cli-tests.sh script also accepts some command-line parameters to adjust its behaviour. You can use the -h option to get additional help on these.

File: src/getting_started/linux_x86.md [hunk 5]
```diff
@@ -75,6 +75,41 @@ MEYCIQCrc9cys35NeXwNAr8lYu8WPu0RiutkoAIWn+jYfYofPwIhAKPlNNsW//ykW8nX11KABNpWWYsN
...
+Note: If `openssl` is not installed globaly in the system.
+`OPENSSL` environment variable need to be set first.
```
Comment (anta5010) [inline src/getting_started/linux_x86.md:111]: If openssl is not installed into a directory included in PATH then you also need to define OPENSSL environment variable with a full path to it:

File: src/getting_started/linux_x86.md (hunk 4 above)
Comment (mohamedasaker-arm) [reply src/getting_started/linux_x86.md:109]: Good idea!

File: src/getting_started/linux_x86.md (hunk 3 above)
Comment (mohamedasaker-arm) [reply src/getting_started/linux_x86.md:90]: Do you think use <full path> here or we can use `$(pwd)`?
as it was used in the previous section [here](https://parallaxsecond.github.io/parsec-book/getting_started/linux_x86.html#using-the-parsec-tool)
just for consistency and I think it's easier for the user to copy and paste.
@anta5010

Comment (anta5010) [reply src/getting_started/linux_x86.md]: Yes, I think we can use $(pwd) in the example as you copy parsec-tool in the same directory in your quickstart script.

File: src/getting_started/linux_x86.md [hunk 6]
```diff
@@ -75,6 +75,52 @@ MEYCIQCrc9cys35NeXwNAr8lYu8WPu0RiutkoAIWn+jYfYofPwIhAKPlNNsW//ykW8nX11KABNpWWYsN
 
 Execute the tool without any argument to check all the possible commands!
 
+## Running the parsec-cli-tests.sh script
+
+It's a script running simple end-two-end Parsec tests using parsec-tool and openssl
```
Comment (ionut-arm) [inline src/getting_started/linux_x86.md:80]:
```suggestion
It's a script running simple end-to-end Parsec tests using parsec-tool and openssl
```

**Review by ionut-arm — APPROVED** [review]
Thanks, looks good!
